sunrise-data-importer/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "sunrise-data-importer",
+  "version": "1.0.0",
+  "description": "",
+  "author": "",
+  "license": "UNLICENSED",
+  "dependencies": {
+    "@commercetools/import-api-ts-sdk": "^0.0.6",
+    "@commercetools/sdk-client": "^1.5.10",
+    "@commercetools/sdk-middleware-auth": "^5.1.3",
+    "@commercetools/sdk-middleware-http": "^4.0.9",
+    "@commercetools/sdk-middleware-queue": "^2.1.1",
+    "@commercetools/typescript-sdk": "^0.0.8-alpha.0",
```

Comment:
Do we really need this?

Reply:
Good find 👍 Fixed here: https://github.com/commercetools/import-api-ts-sdk/pull/4/commits/4bd52c61c49a9624dd13809b7972c34770cb3853